query-engine/black-box-tests/tests/helpers/mod.rs

```diff
@@ -17,8 +17,8 @@ where
 
     let mut child = command
         .stdin(std::process::Stdio::piped())
-        .stdout(std::process::Stdio::piped())
-        .stderr(std::process::Stdio::null())
+        .stdout(std::process::Stdio::inherit())
```

Comment:
I was thinking about removing those, but they turned out to be very helpful during debugging - maybe we can leave it?

Reply:
Let's leave it for now 👍